locale/en/auth.ftl

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Hmm I'm guessing this can just be an exact copy of `templates/auth.ftl`, will update next week.

Reply:
If en-US contains unique string specific to US market, then make `en` the source for localization, and `en-US` is just a locale

Reply:
Since our default is en-US, I think that means we don't need a file there at all then?

Reply:
If you look at the other files, all have an `en` or `template` file that contains the source string. Right now, to see what's in this file, I have to refer to en-CA or en-GB instead. Having said that, in the en or en-US file, you have different set of strings, all placeholders.